.github/workflows/verify.yaml

```diff
@@ -49,6 +49,31 @@ jobs:
       - name: Dependency analysis report
         run: cat build/reports/dependency-analysis/build-health-report.txt
 
+  OpenAPI-Definitions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '17'
+
+      - name: Save previous OpenAPI definition
+        run: yq -P 'sort_keys(..)' resources/openapi/openapi.yaml > openapi-git-sorted.yaml
```

Comment:
Is yq installed by default on the ubuntu-latest distro?

Reply:
Yes, as demonstrated by the build respectively succeeding, or failing with diff